packages/mobile/src/components/core/SectionList.tsx

```diff
@@ -53,25 +51,28 @@ const CollapsibleSectionList = ({
 
   return (
     <View>
-      <Portal hostName='PullToRefreshPortalHost'>
-        <PullToRefresh
-          isRefreshing={refreshing}
-          onRefresh={onRefresh}
-          scrollAnim={collapsibleScrollAnim}
-          topOffset={40}
-          color={staticWhite}
-        />
-      </Portal>
+      {onRefresh ? (
+        <Portal hostName='PullToRefreshPortalHost'>
+          <PullToRefresh
+            isRefreshing={refreshing}
+            onRefresh={onRefresh}
+            scrollAnim={collapsibleScrollAnim}
+            topOffset={40}
+            color={staticWhite}
+          />
+        </Portal>
+      ) : null}
       <Animated.SectionList {...other} {...scrollPropsAndRef} />
     </View>
   )
 }
 
 const AnimatedSectionList = forwardRef<RNSectionList, SectionListProps>(
   function AnimatedSectionList(
-    { refreshing, onRefresh, ...other },
+    props,
     ref: MutableRefObject<RNSectionList<any, DefaultSectionT> | null>
   ) {
+    const { refreshing, onRefresh, ...other } = props
```

Comment:
Why prefer destructuring in the function body?

Reply:
i like how it cleans up the `const Blah = () => ` part of things. Also sometimes I've had cases where we want the full `props` object and just need to reference a part of props in body, sometimes destructure some and create ...other, so this way it can be the same approach, if that makes sense? it also cleans up docs: 
<img width="659" alt="image" src="https://user-images.githubusercontent.com/8230000/162096214-ae3ee2d7-ba4a-4151-b294-dbe3fac4bcdc.png">
<img width="749" alt="image" src="https://user-images.githubusercontent.com/8230000/162096230-e82cbfed-1f3a-4b64-b105-594cd7a924bd.png">

Reply:
Yeah I definitely have run into the case of needing the whole props object, I think being consistent make sense